services/graphql-server/src/graphql/resolvers/platform/content.js

```diff
@@ -49,7 +49,10 @@ const retrieveYoutubePlaylistId = async ({ youtube }) => {
     payload.forUsername = forUsername;
   }
   const response = await googleDataApiClient.request('youtube.channelList', payload);
-  return get(response, 'items.0.contentDetails.relatedPlaylists.uploads');
+  if (get(response, 'items.0.contentDetails.relatedPlaylists.uploads')) {
+    return get(response, 'items.0.contentDetails.relatedPlaylists.uploads');
+  }
+  return '';
```

Comment:
@Shinsina - the title mentions returning an empty array, but this is returning an empty string...is that correct in this case?

Reply:
@brandonbk This particular resolver/function returns the id of a playlist as a string as far as I know. The title of the PR is accurate in that what is actually being returned via API call resolution to the front-end Vue component will fall onto an empty array should the information be invalid.